code/testapp-kotlin/src/main/java/com/adobe/marketing/mobile/app/kotlin/MyApp.kt

```diff
@@ -22,9 +24,9 @@ class MyApp : Application() {
         super.onCreate()
         MobileCore.setApplication(this)
         MobileCore.setLogLevel(LoggingMode.VERBOSE)
-//        MobileCore.configureWithAppID("your app id")
-
-        MobileCore.registerExtensions(listOf(Identity.EXTENSION, PerfExtension::class.java)) {
+        // MobileCore.configureWithAppID("YOUR_APP_ID")
+        val extensions = listOf(Identity.EXTENSION, Signal.EXTENSION, Lifecycle.EXTENSION, PerfExtension::class.java)
```

Comment:
Nit: Can we do 
```
val APP_ID = "YOUR_APP_ID"
..
..
MobileCore.configureWithAppID(APP_ID)
```
So that we can retain the configureWithAppId without commenting it out.

Reply:
The application contains a local config file. I prefer keeping this line commented to avoid additional configuration event. 
We can follow your pattern in the sample app repository.